website/docs/r/vapp_vm.html.markdown

```diff
@@ -334,6 +334,8 @@ example for usage details.
 ## Attribute reference
 
 * `vm_type` (*3.2+*) - type of the VM (either `vcd_vapp_vm` or `vcd_vm`)
+* `status` - (*v3.8+*) The vApp status as a numeric code
+* `status_text` - (*v3.8+*) The vApp status as text.
```

Comment:
Nit: the "." is inconsistent with above bullet points.

Reply:
True, I gave tried to align with adjacent fields.